INSTALL

```diff
@@ -67,6 +67,20 @@ frame pointer based stack unwinder by passing the
 --enable-frame-pointers flag to configure.
 
 
+Building glog - Using vcpkg
+===========================
+
+You can download and install glog using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    ./vcpkg install glog
+
+The glog port in vcpkg is kept up to date by Microsoft team members and community contributors. If the version is out of date, please [create an issue or pull request](https://github.com/Microsoft/vcpkg) on the vcpkg repository.
```

Comment:
The sentence exceeds the maximum number of characters used in this file. Either format the sentence accordingly or convert the file to markdown. The latter would fix the issues with your links.